packages/app/src/cli/models/templates/ui-specifications/flow_action_definition.ts

```diff
@@ -0,0 +1,27 @@
+import {ExtensionTemplate} from '../../app/template.js'
```

Comment:
Quick question: do these need to be located in the `ui-specifications` folder and if yes will that change in the future or is this how generic extensions specs will always be created?

Reply:
no, no need to have it here, forgot to move these, we can move them outside this folder.
Not that it's relevant anyway, is just for us to organize code internally

Reply:
Ah I see the import below. Any objections to moving this to a `config-only-specifications` or `generic-specifications` folder?

Reply:
Sounds good, thank you!